### PR TITLE
foreach loop should use list()

### PR DESCRIPTION
**Pull request by KeremArdicli**
Also added extra information about the error to the catch block

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: docs/en/queries/transactions.md [hunk 1]
```diff
@@ -38,12 +38,14 @@ try {
         [3, 'db'],
     ];
         
-    foreach ($tags as [$id, $name] => $tag) {
+    foreach ($tags as list($id, $name)) {
```
Comment (samdark) [inline docs/en/queries/transactions.md:41]: Why? It's the same, isn't it?

File: docs/en/queries/transactions.md [hunk 2]
```diff
@@ -38,12 +38,14 @@ try {
...
         $insertTagCommand->execute();
         $insertPostTagCommand->execute();
     }    
-} catch () {
+} catch (Exception $e) {
     $transaction->rollBack();
+    // to get a slightest info about the Exception
+    var_dump($e->getMessage());
```
Comment (samdark) [inline docs/en/queries/transactions.md:48]: That's a no go. Echoing exception isn't a good idea.

File: docs/en/queries/transactions.md (hunk 1 above)
Comment (KeremArdicli) [reply docs/en/queries/transactions.md:41]: On my local env with win10, apache, PHP 8.1.1 it throws error. list() does not.

File: docs/en/queries/transactions.md (hunk 2 above)
Comment (KeremArdicli) [reply docs/en/queries/transactions.md:48]: Yeah maybe commenting it and saying to use it only for development would be needed. It is a no go for prod.

**Comment by samdark** [comment]
Thanks.

**Pull request merged**